.github/workflows/base.yml

```diff
@@ -18,10 +18,22 @@ jobs:
             cuda_version: 11.8.0
             cuda_version_bnb: "118"
             pytorch: 2.0.0
+            axolotl_extras:
           - cuda: cu117
             cuda_version: 11.7.0
             cuda_version_bnb: "117"
             pytorch: 1.13.1
+            axolotl_extras:
+          - cuda: cu118
```

Comment:
I think cuda and cuda_version_bnb can be combined. In the image, you can just add “cu” in front of the cuda version.